htdocs/web_portal/static_html/Help_And_Contact.html

```diff
@@ -6,49 +6,52 @@ <h1>Documentation, Help and Support</h1>
        <h2>GOCDB</h2>
         <p><a href="index.php">What is GOCDB?</a></p>
         <p>
-         See <a href="https://github.com/GOCDB/gocdb/blob/master/changeLog.txt">Release notes</a>
+         See <a href="https://github.com/GOCDB/gocdb/releases">Release notes</a>
         <br/>
-        <a href="https://wiki.egi.eu/wiki/GOCDB/Regional_Module_Technical_Documentation">Download, Install</a>
+        <a href="https://github.com/GOCDB/gocdb/blob/dev/INSTALL.md">Download, Install</a>
         </p>
              
     <h2>Need information?</h2> 
     <p>
-      Please browse  
-      <a href="https://wiki.egi.eu/wiki/GOCDB">
-      GOCDB Documentation Index</a> on the EGI wiki.
+      Please browse the GOCDB 
+      <a href="https://gocdb.github.io/">
+      Documentation </a> or your community documentation source.
       <br/><br/>
     </p>
     
     <h2>Found a bug?</h2> 
     <p> 
-      First, <a href='https://rt.egi.eu/rt/Dashboards/5541/GOCDB-Requirements' target='_blank'>
-      check known bugs and feature requests</a> to see if this has not 
-      already been reported.<br/> 
-      If not, please raise a ticket against GOCDB in
-      <a href='https://ggus.eu' target='_blank'> 
-      GGUS.</a>
+      First, check our 
+      <a href='https://github.com/GOCDB/gocdb/issues?q=is%3Aissue+is%3Aopen+label%3Abug' target='_blank'>
+      known bugs</a> to see if this has not already been reported.<br/> 
+      If not, please open a request in your community support desk or our 
+      <a href='https://github.com/GOCDB/gocdb/issues' target='_blank'> 
+      GitHub issues </a> 
```

Comment:
```suggestion
      GitHub issues</a> 
```

Reply:
yeah i had already seen this problem and edited locally